addOns/fuzz/src/main/javahelp/org/zaproxy/zap/extension/fuzz/resources/help/contents/api.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+    <META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+    <TITLE>
+        Fuzzing API
+    </TITLE>
+</HEAD>
+<H1>Fuzzing API</H1>
+<p>
+    The API has been designed to be used by people with a range of security experience, for
+    example “simpleHttpFuzzer” is a basic fuzzer fuzzes one “Fuzz location”, extracting payloads
+    from one file, and returns the fuzzerId.
+</p>
+<h3>Homepage</h3>
+<p><a href="https://github.com/zaproxy">https://github.com/zaproxy</a><br>
+    Documentation can be found at:<br>
+    <a href="https://github.com/zaproxy/zap-api-docs/fuzz">https://github.com/zaproxy/zap-api-docs/fuzz</a>
+    <br>
+</p>
+<h3>Message Processors</h3>
+<p>
+    The default processor enabled by default for all requests is “Request Content-Length Updater”
```

Comment:
Should this be "default" or "mandatory" as I recall the earlier code the user isn't able to unset it.

Reply:
True, the user is not able to unset it. Changing it to The requisite processor.